### PR TITLE
Remove extraneous trailing whitespace

### PR DESCRIPTION
**Pull request by jlapeyre**

### CODE REVIEW

File: test/test_qubitarrays.jl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using PastaQ
+xusing PastaQ
```
Comment (mtfishman) [inline test/test_qubitarrays.jl:1]: Looks like a mistake here?

**Comment by mtfishman** [comment]
Thanks John, I guess the code formatter ignores that issue in comments. Maybe there is a setting for that?

File: test/test_qubitarrays.jl (hunk 1 above)
Comment (jlapeyre) [reply test/test_qubitarrays.jl:1]: Yes. I'm away, but will fix it soon.I was tired and working with a different setup. Had a feeling something might be incorrect.

**Comment by jlapeyre** [comment]
You mean it ignores extraneous spaces?

**Comment by mtfishman** [comment]
I think the code formatting ignores comments entirely, which would explain why these trailing spaces weren't caught already.

File: test/test_randomstates.jl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-using PastaQ
+xusing PastaQ
```
Comment (mtfishman) [inline test/test_randomstates.jl:1]: Same issue.

File: test/test_runcircuit.jl [hunk 3]
```diff
@@ -1,4 +1,4 @@
-using PastaQ
+xusing PastaQ
```
Comment (mtfishman) [inline test/test_runcircuit.jl:1]: Same issue.

Comment (jlapeyre) [reply test/test_runcircuit.jl:1]: Arggg. I grepped for this now and removed them.

**Comment by mtfishman** [comment]
Thanks John!

**Pull request merged**